lms/static/certificates/sass/_components.scss

```diff
@@ -96,27 +96,27 @@
 
 // message to user-centric banner
 .wrapper-banner-user {
-    box-shadow: inset 0 rem(2) rem(2) 0 palette(grayscale, black-t);
-    background: palette(grayscale-cool, x-dark);
+    box-shadow: inset 0 rem(2) rem(2) 0 palette(grayscale, black);
```

Comment:
I think the original was pure black, so I've made it the same. I'm starting to think that we need variables for certificates colors so that it is easier to see the intent, and also to support theming. I'm not going to do that here though...